src/query/query_hash_scan.h

```diff
@@ -77,17 +90,41 @@ struct hash_scan_key
   db_value **values;		/* value array */
 };
 
+typedef struct file_hash_scan_id FHSID;
+struct file_hash_scan_id
+{
+  /* Fields should be ordered according to their sizes */
+  EHID ehid;			/* dir file identifier */
+  VFID bucket_file;		/* bucket file identifier */
+  DB_TYPE key_type;		/* type of the keys */
+  short depth;			/* global depth of the directory */
```

Comment:
how about unsigned short or unsigned char ?

Reply:
I fixed it.